### ISSUE TITLE
'Co presenter' role needed when using the 'Camera as Content' feature or 'polls for moderators' instead of presenters)

### ISSUE THREAD

**Issue by simpannar** [bigbluebutton/bigbluebutton#19398]
We love the new 'Camera as Content' feature introduced in BBB v2.7. However, the current implementation comes with a downside. Let me first describe our learning scenario, before I describe my suggestions to fix this.

- We mainly use BBB for all-day or even multi-day online or hybrid seminars in the field of Smart Home and Building Technology.
- In these trainings, certain technical setups have to be shown in detail on a regular basis. 
- We do the trainings in groups between 10 and 100 participants. 
- We try to engage the audience as much as possible (hands-on exercises, polls, discussions etc.)
- When dealing with larger groups, sometimes we conduct the seminars with more than one trainer (trainer + co-trainer so to speak). 

In order to optimize the user experience, we use a streaming setup with multiple cameras and a video switcher. This allows us to easily switch between the 

1. main cameras (trainer/s / moderators in the training room)
2. detail cameras (for the demo / lab setups) and
3. the presentations /software demos on the trainer notebooks

The new 'Camera as Content' feature seamed perfect for our scenario, because we can provide our stream coming from the video switcher in the content section which highlights this stream while all the webcams of our participants can be presented in smaller parts of the browser window (we use the 'smart layout' in push that layout to our participants).  

Please note that the video stream which is used as the 'content cam' does not necessarily come from the computer of the trainer, but from a dedicated 'streaming PC'. 
This 'streaming PC' logs into the BBB room, gets presenter privileges and starts the stream with the 'Camera as content' feature. But this streaming PC is only a 'technical' user. Especially when we conduct hybrid trainings the streaming PC may not even be nearby the speaker's desk, but maybe at the opposite side of our classroom.

**Downside:**

- Our trainers join the room, too. 
- They use a second notebook positioned at their speaker's desk to manage the online audience (manage chat, create polls, creating breakout-rooms, etc.)
- For instance, we use polls in our trainings in many situations as a means to engage the online audience... 

However, since the 'streaming PC' is the 'Presenter' in BBB our trainers cannot start a poll. If they take presenter privileges - this would cause the 'camera as content' coming from the streaming PC to be stopped.... The same goes for the 'Select a random user' feature. 

 
**Describe the solution you'd like**

- Easiest solution would be to give the 'start the poll' features to any user which joins the session as 'moderator'. 
- The same goes for the 'select random user'. 

I see no reason anymore why 'start a poll' should be restricted to the user who is presenter.
In ancient versions of BBB the results of polls have been displayed on top of the presentation area. However, today the poll results are pushed to the public chat area and the current editor for polls is very powerful which means that someone who is not presenting can easily prepare and initiate the poll and publish the results while someone else is doing the presentation....

This relatively small change allows a better allocation of tasks between presenters (keynote speakers / instructors etc.) and additional moderators ('co-trainers') - regardless if the 'camera as content' feature is used or not. Because the limitation described above also applies if the main trainer is sharing his screen via 'desktop sharing' and a second trainer should act as a 'co-trainer'.

In other words: small change, much more flexibility when the event is managed by a small team of trainers / moderators. 

I hope this description was clear enough. 
Looking forward to additional feedback. 

Thank you so much in advance!
